package.json

```
@@ -292,7 +292,7 @@
 		"storybook:dev": "concurrently \"npm run dev:packages\" \"start-storybook -c ./storybook -p 50240\"",
 		"test": "npm-run-all lint test:unit",
 		"test:create-block": "bash ./bin/test-create-block.sh",
-		"test:e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
+		"test:e2e": "wp-scripts test-e2e packages/e2e-tests/specs/editor/various/draggable-block.test.js --config packages/e2e-tests/jest.config.js",
```

Let's restore the original script

```suggestion
		"test:e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
```